vault/resource_identity_group.go

```diff
@@ -53,6 +55,13 @@ func identityGroupResource() *schema.Resource {
 				Description: "Policies to be tied to the group.",
 			},
 
+			"external_policies": {
```

Comment:
This field seems a bit counter-intuitive if you compare it with the `google_xxx_iam_*` or other similar resources.

May I suggest that:

- if `policies` is not provided, do a [`DiffSuppressFunc`](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#diffsuppressfunc) to suppress a diff on `policies`
- Likewise, do not set `policies` while writing the entity.

Then, users can use the new resources you have added to manage policies.

Reply:
Did you miss out a `resource_identity_policies.go`?

Reply:
I don't think it is possible to tell whether the value was set by user or read from the Vault.